mopidy_pandora/actor.py

```diff
@@ -121,13 +122,16 @@ class TrackUri(StationUri):
 class PandoraLibraryProvider(backend.LibraryProvider):
     root_directory = models.Ref.directory(name='Pandora', uri=PandoraUri('directory').uri)
 
-    def __init__(self, backend):
+    def __init__(self, backend, sort_order):
+        self.sort_order = sort_order
```

Comment:
Good point, I've made the optimisation and committed to https://github.com/jcass77/mopidy-pandora/commit/33bb88b1ec95bdf232d3d42258597827ccc28666